### DISCUSSION TITLE
Sort list of current applications by date added

### DISCUSSION THREAD

**Discussion by justsolomon** [jenndryden/Canadian-Tech-Internships-and-New-Grad-2025#113] [Ideas]
Hi there!👋🏾

I came across this repo when I was searching for internships specifically for Canadian students, and I want to start off by saying a huge thank you for creating this!

Regarding the discussion title, I think adding the date the opening was added to the list would be beneficial, similar to how it is on the [Pitt CSC repo](https://github.com/SimplifyJobs/Summer2024-Internships), so the list can be somewhat ordered from the latest date posted to the earliest. I also think it'll help reduce confusion because the unordered format kind of makes it a bit difficult to track which of the internships are new. For example, the openings for Harris Computer Systems and Intel were added on the same day, but HCS is at the bottom of the list, while Intel is at the top.

If this sounds like a good idea, I can make a PR to add the dates posted for the current applications list using the dates the commits were merged to the `master` branch. Subsequent PRs would then include the date posted and be added at the top of the list. I can also update the template and the instructions for the contributing guidelines in the same PR.

**Comment by jenndryden** [comment]
This is a fantastic idea, go for it!! And i’m glad it has been helpful for you 😄 Thanks for your input
